src/build_options.rs

```diff
@@ -587,6 +591,16 @@ impl BuildOptions {
             self.find_interpreters(&bridge, &interpreter, &target, None, generate_import_lib)?
         };
 
+        if !self.allow_prereleases {
+            interpreter.retain(|interp| {
+                let version = Version::new([interp.config.major as u64, interp.config.minor as u64]);
```

Comment:
This won't be enough, I think it also need patch version.

Reply:
```
Python 3.13.0a6 (main, Apr 10 2024, 01:20:43) [GCC 12.2.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import sys
>>> sys.version_info
sys.version_info(major=3, minor=13, micro=0, releaselevel='alpha', serial=6)
>>> sys.version
'3.13.0a6 (main, Apr 10 2024, 01:20:43) [GCC 12.2.0]'
```

Reply:
ok

Reply:
interp config only has major and minor, missing micro，we should pass minor to it?

Reply:
You can modify interp config to add more detailed version info.

Reply:
Perhaps we can remove `major` and `minor`, add `version: pep440_rs::Version` instead to interpreter config and maybe other places. I might give it a try on the weekend.

Reply:
ok